examples/kubernetes/glusterd-ds.yaml

```diff
@@ -18,69 +18,69 @@ spec:
     spec:
       hostNetwork: true
       containers:
-      - image: docker.io/gluster/glusterd2-nightly
-        imagePullPolicy: IfNotPresent
-        name: glusterfs
-        resources:
-          requests:
-            memory: 100Mi
-            cpu: 100m
-        volumeMounts:
+        - image: docker.io/gluster/glusterd2-nightly
+          imagePullPolicy: IfNotPresent
+          name: glusterfs
+          resources:
+            requests:
+              memory: 100Mi
+              cpu: 100m
+          volumeMounts:
+            - name: glusterfs-run
+              mountPath: "/run"
+            - name: glusterfs-lvm
+              mountPath: "/run/lvm"
+            - mountPath: /var/lib/glusterd2
+              name: glusterd2-config
+            - name: glusterfs-etc
+              mountPath: "/etc/glusterfs"
+            - name: glusterfs-logs
+              mountPath: "/var/log/glusterfs"
+            - name: glusterfs-config
+              mountPath: "/var/lib/glusterd"
+            - name: glusterfs-dev
+              mountPath: "/dev"
+            - name: glusterfs-misc
+              mountPath: "/var/lib/misc/glusterfsd"
+            - name: glusterfs-cgroup
+              mountPath: "/sys/fs/cgroup"
+              readOnly: true
+            - name: glusterfs-ssl
+              mountPath: "/etc/ssl"
+              readOnly: true
+            - name: kernel-modules
+              mountPath: "/usr/lib/modules"
+              readOnly: true
+          securityContext:
+            capabilities: {}
+            privileged: true
+      volumes:
         - name: glusterfs-run
-          mountPath: "/run"
+        - name: glusterd2-config
         - name: glusterfs-lvm
-          mountPath: "/run/lvm"
-        - mountPath: /var/lib/glusterd2
-          name: glusterd2-config
+          hostPath:
+            path: "/run/lvm"
         - name: glusterfs-etc
-          mountPath: "/etc/glusterfs"
+          hostPath:
+            path: "/etc/glusterfs"
         - name: glusterfs-logs
-          mountPath: "/var/log/glusterfs"
+          hostPath:
+            path: "/var/log/glusterfs"
         - name: glusterfs-config
-          mountPath: "/var/lib/glusterd"
+          hostPath:
+            path: "/var/lib/glusterd"
         - name: glusterfs-dev
-          mountPath: "/dev"
+          hostPath:
+            path: "/dev"
         - name: glusterfs-misc
-          mountPath: "/var/lib/misc/glusterfsd"
+          hostPath:
+            path: "/var/lib/misc/glusterfsd"
         - name: glusterfs-cgroup
-          mountPath: "/sys/fs/cgroup"
-          readOnly: true
```

Comment:
@JohnStrunk why `RO` is removed here ? Are all functionalities working ?  we had kept it based on recommendation when systemd is in use inside the container.

Reply:
@humblec `RO` is not removed here, its problem with Github diff i think
check `volumeMounts` section in yaml `RO` still exists.

Reply:
No changes other than indenting were made to these files.